ui/tests/acceptance/secure-variables-test.js

```diff
@@ -146,4 +152,67 @@ module('Acceptance | secure variables', function (hooks) {
     await Variables.visit();
     await a11yAudit(assert);
   });
+
+  module('create flow', function () {
+    test('allows a user with correct permissions to create a secure variable', async function (assert) {
+      // Arrange Test Set-up
+      defaultScenario(server);
+      server.createList('variable', 3);
+      const variablesToken = server.db.tokens.find(SECURE_TOKEN_ID);
+      window.localStorage.nomadTokenSecret = variablesToken.secretId;
+      await Variables.visit();
+      // End Test Set-up
+
+      assert
+        .dom('[data-test-create-var]')
+        .exists(
+          'It should display an enabled button to create a secure variable'
+        );
+      await click('[data-test-create-var]');
+
+      assert.equal(currentRouteName(), 'variables.new');
+
+      await typeIn('[data-test-path-input]', 'foo/bar');
+      await typeIn('[data-test-var-key]', 'kiki');
+      await typeIn('[data-test-var-value]', 'do you love me');
+      await click('[data-test-submit-var]');
+
+      assert.equal(
+        currentRouteName(),
+        'variables.variable.index',
+        'Navigates user back to variables list page after creating variable.'
+      );
+      assert
+        .dom('.flash-message.alert.alert-success')
+        .exists('Shows a success toast notification on creation.');
+      assert
+        .dom('[data-test-var=kiki]')
+        .exists('The new variable key should appear in the list.');
+
+      // Reset Token
+      window.localStorage.nomadTokenSecret = null;
+    });
+
+    test('prevents users from creating a secure variable without proper permissions', async function (assert) {
+      // Arrange Test Set-up
+      defaultScenario(server);
+      server.createList('variable', 3);
+      const variablesToken = server.db.tokens.find(SECURE_TOKEN_ID);
+      window.localStorage.nomadTokenSecret = variablesToken.secretId;
+      const policy = server.db.policies.find('Variable Maker');
+      policy.rulesJSON.Namespaces[0].SecureVariables['Path "*"'].Capabilities =
+        ['list'];
+      await Variables.visit();
+      // End Test Set-up
+
+      assert
+        .dom('[data-test-disabled-create-var]')
+        .exists(
+          'It should display an enabled button to create a secure variable on the main listings page'
```

Comment:
replace "enabled" with "disabled" here; 

alternatively: you can test for [isDisabled](https://github.com/simplabs/qunit-dom/blob/master/API.md#isdisabled) here